runtime/onert/core/include/backend/IConfig.h

```diff
@@ -32,20 +32,43 @@ namespace backend
 struct IConfig
 {
   virtual ~IConfig() = default;
-
+  /**
+   * @brief Returns ID of the backend
+   *
+   * @return std::string ID of this backend
+   */
   virtual std::string id() = 0;
+  /**
+   * @brief Initialize the backend. This is called as soon as the backend is loaded.
+   *
+   * @return true  Initialization succeeded
+   * @return false Initialization failed, so it cannot use this backend
+   */
   virtual bool initialize() = 0;
-  // Support permute kernel
-  virtual bool supportPermutation() = 0;
+  /**
+   * @brief Returns supported layout for the given \p node and \p frontend_layout
+   *
+   * @param node Operation
+   * @param frontend_layout The layout defined in the model
+   * @return ir::Layout The layout that the backend kernel actually uses
+   */
   virtual ir::Layout supportLayout(const ir::Operation &node, ir::Layout frontend_layout) = 0;
```

Comment:
Does somebody know the use of `const ir::Operation &node`? It seems like it is never used in any implementations.